### PR TITLE
Reformat examples using black

### PR DESCRIPTION
**Pull request by virtuald**
It doesn't look so bad.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -9,6 +9,17 @@ install:
 # command to run tests
 script:
   - ./run_tests.sh all
+jobs:
+  include:
+  - stage: format-check
+    sudo: false
+    language: python
```
Comment (auscompgeek) [inline .travis.yml]: `sudo` and `language` should be inherited from the top-level, right?

File: .travis.yml [hunk 2]
```diff
@@ -9,6 +9,17 @@ install:
 # command to run tests
 script:
   - ./run_tests.sh all
+jobs:
+  include:
+  - stage: format-check
```
Comment (auscompgeek) [inline .travis.yml]: I feel this should be called `lint` instead. What if we want to run some more checks in parallel in future? (pyflakes in particular comes to mind.)

File: navx-rotate-to-angle-arcade/physics.py [hunk 3]
```diff
@@ -1,35 +1,34 @@
...
-            32*units.inch + bumper_width*2,     # robot length
-            6*units.inch                        # wheel diameter
+            motor_cfgs.MOTOR_CFG_CIM,  # motor configuration
+            110 * units.lbs,  # robot mass
+            10.71,  # drivetrain gear ratio
+            2,  # motors per side
+            22 * units.inch,  # robot wheelbase
+            23 * units.inch + bumper_width * 2,  # robot width
+            32 * units.inch + bumper_width * 2,  # robot length
+            6 * units.inch,  # wheel diameter
```
Comment (auscompgeek) [inline navx-rotate-to-angle-arcade/physics.py]: Hmm. Would using keyword arguments make this more readable?

File: .travis.yml (hunk 2 above)
Comment (virtuald) [reply .travis.yml]: pyflakes is more of a linter. This doesn't check for any kind of correctness, it just formats. Thus, format-check.

Comment (auscompgeek) [reply .travis.yml]: Sure, but I don't think we should have any of these sorts of checks block any of the others from running in CI.

Comment (virtuald) [reply .travis.yml]: Doesn't this run last?

Comment (auscompgeek) [reply .travis.yml]: Yeah, but I'm suggesting we might want to run pyflakes alongside black in future. Or maybe even now actually. Seems like a good idea at least.

File: navx-rotate-to-angle-arcade/physics.py (hunk 3 above)
Comment (virtuald) [reply navx-rotate-to-angle-arcade/physics.py]: Oh, I see what you mean. Yeah, that might help.

Comment (virtuald) [reply navx-rotate-to-angle-arcade/physics.py]: It's not really better. It's probably best to just disable formatting for that.

**Pull request merged**